types/splitio.d.ts

```diff
@@ -559,6 +571,39 @@ declare namespace SplitIO {
    */
   interface SplitToGaIntegration {
     type: 'SPLIT_TO_GA',
+    /**
+     * Optional filter to use instead of default, which always return true, 
+     * meaning that all impressions and events are tracked as GA hits. 
+     */
+    filter?: (data: SplitIO.Data) => boolean,
```

Comment:
The name "SplitIO.Data" is too generic. This is not Split data as a whole. Should be IntegrationData or something like that.